tests/cli/lib/lib.sh

```diff
@@ -47,7 +47,7 @@ check_root_account() {
         # ssh returns 255 in case of any ssh error, so it's better to grep the specific error message
         rlRun -t -c "ssh $SSH_OPTS -o PubkeyAuthentication=no root@${SSH_MACHINE} 2>&1 | grep -i 'permission denied ('" \
             0 "Can't ssh to '$SSH_MACHINE' as root using password-based auth"
-        rlRun -t -c "ssh $SSH_OPTS ${SSH_USER}@${SSH_MACHINE} \"sudo grep -E '^root:(\*LOCK\*|!):' /etc/shadow\"" \
+        rlRun -t -c "ssh $SSH_OPTS ${SSH_USER}@${SSH_MACHINE} \"sudo grep -E '^root:.*[:;\*!\\].*(:.*){7}' /etc/shadow\"" \
```

Comment:
Checking with a regular expression seems error-prone to me. You can run
```
passwd --status root
```
which outputs  `L` or `LK` as its second field for locked accounts. I wouldn't block this PR based on that. Just FYI :)

Reply:
Thanks for pointing this out, I changed the check to use `passwd --status`, it should be more future-proof :).